src/api.v2/model/userAccess.js

```diff
@@ -0,0 +1,19 @@
+const createGuts = require('../helpers/modelGuts');
+
+const tableName = 'user_access';
+
+const selectableProps = [
+  'user_id',
+  'experiment_id',
+  'access_role',
+  'updated_at',
+];
+
+const guts = createGuts({
+  tableName,
+  selectableProps,
+});
+
+module.exports = {
+  ...guts,
+};
```

Comment:
Are we spreading because we want a new instace of the object?

Reply:
we are spreading to get all the properties of the object, not the object itself